doc/api/index.md

```diff
@@ -35,7 +34,8 @@
 * [HTTPS](https.html)
 * [Inspector](inspector.html)
 * [Internationalization](intl.html)
-* [Modules](modules.html)
+* [Modules: CommonJS modules](modules.html)
```

Comment:
```suggestion
* [Modules: CommonJS](modules.html)
```

Reply:
```suggestion
* [Packages: CommonJS modules](modules.html)
```

Would that work?

Reply:
Historically we've always referred to this as modules. I don't think many users will think to look for this under Packages.

I think `Modules: CommonJS modules` and `Modules: ECMAScript modules` are the best names. @Trott was your objection just to the repetition of the word, or was there something else?